app/src/Tasks/index.js

```diff
@@ -17,40 +15,43 @@ const Tasks = () => {
   const [threads, setThreads] = React.useState([]);
   const [meditation, setMeditation] = React.useState([]);
   const [quote, setQuote] = React.useState({});
+  const [pairs, setPairs] = React.useState({});
```

Comment:
if you were expecting `pairs` to be an array, need to set it as `React.useState([])`